homeassistant/components/notion/__init__.py

```diff
@@ -290,15 +290,20 @@ def __init__(
         """Initialize the entity."""
         super().__init__(coordinator)
 
-        bridge = self.coordinator.data.bridges[bridge_id]
+        if bridge := self._async_get_bridge(bridge_id):
+            bridge_hardware_id = bridge.hardware_id
+        else:
+            bridge_hardware_id = ""
+
         sensor = self.coordinator.data.sensors[sensor_id]
+
         self._attr_device_info = DeviceInfo(
             identifiers={(DOMAIN, sensor.hardware_id)},
             manufacturer="Silicon Labs",
             model=str(sensor.hardware_revision),
             name=str(sensor.name).capitalize(),
             sw_version=sensor.firmware_version,
-            via_device=(DOMAIN, bridge.hardware_id),
+            via_device=(DOMAIN, bridge_hardware_id),
```

Comment:
So now bridge_hardware_id can be "". Will there also be a device with empty id? Should we not add via_device when this happens?